install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -58,6 +58,27 @@ where:
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return user specify priorityClass or default criticalPriorityClass
```

Comment:
I just added the new helper because it looked better for me. But indeed those helpers are identical, i will remove the spire helper again and use the cilium helper instead.

Reply:
@gandro Now the cilium helper gets used again. Hope this looks better now :)

Reply:
Great, thank you!